internal/admin/dashboard/templates/index.html

```
@@ -1223,190 +1223,190 @@ <h2>Workflows</h2>
                     </div>
 
-                    <div class="execution-plan-guardrail-list-editor" x-show="executionPlanForm.guardrails.length > 0">
-                        <template x-for="(step, index) in executionPlanForm.guardrails" :key="'execution-plan-form-guardrail-' + index">
-                            <div class="execution-plan-guardrail-row">
-                                <input type="text" class="filter-input execution-plan-input" list="execution-plan-guardrail-options" placeholder="Guardrail ref" x-model="step.ref" :aria-label="'Guardrail reference ' + (index + 1)">
-                                <input type="number" class="filter-input execution-plan-step-input" min="0" step="10" placeholder="Step" x-model.number="step.step" :aria-label="'Guardrail step ' + (index + 1)">
-                                <button type="button" class="table-action-btn table-action-btn-danger" @click="removeExecutionPlanGuardrailStep(index)">Remove</button>
+                    <div class="workflow-guardrail-list-editor" x-show="workflowForm.guardrails.length > 0">
+                        <template x-for="(step, index) in workflowForm.guardrails" :key="'workflow-form-guardrail-' + index">
+                            <div class="workflow-guardrail-row">
+                                <input type="text" class="filter-input workflow-input" list="workflow-guardrail-options" placeholder="Guardrail ref" x-model="step.ref" :aria-label="'Guardrail reference ' + (index + 1)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)